src/views/lighttable.c

```diff
@@ -2723,6 +2767,9 @@ static void drag_and_drop_received(GtkWidget *widget, GdkDragContext *context, g
 // shitf the first select image by 1 with up direction
 static void shift_first_selected_image(dt_library_t *lib, const int up)
 {
+  // reset the culling layout
+  _expose_destroy_slots(darktable.view_manager->proxy.lighttable.view);
```

Comment:
This is not used by culling.

Reply:
yes it is ;) afaik, it's called when you press arrow keys (see key_pressed)

Reply:
Ups! My bad!